src/Theme/FooterTemplate.elm

```diff
@@ -15,10 +15,14 @@ view model =
             translate model.language
     in
     footer [ css [ footerStyle ] ]
-        [ h2
+        [ img [ src "/images/dfl-logo.svg" ] []
```

Comment:
In the design, the text sits on top of the logo. Also see note about either using a dummy logo or one from their website or Emma.